app/__init__.py

```diff
@@ -33,18 +76,12 @@ def create_app(config_name):
         search_api_client=search_api_client,
     )
 
-    frameworks = data_api_client.find_frameworks().get('frameworks')
-    for framework_data in frameworks:
-        if not framework_data['slug'] in application.config.get('DM_FRAMEWORK_CONTENT_MAP', {}):
-            if framework_data['framework'] == 'g-cloud':
-                content_loader.load_manifest(framework_data['slug'], 'services', 'services_search_filters')
-                # we need to be able to display old services, even on expired frameworks
-                content_loader.load_manifest(framework_data['slug'], 'services', 'display_service')
-                content_loader.load_manifest(framework_data['slug'], 'services', 'download_results')
-                try_load_metadata(content_loader, application, framework_data, ['following_framework'])
-            elif framework_data['framework'] == 'digital-outcomes-and-specialists':
-                content_loader.load_manifest(framework_data['slug'], 'briefs', 'display_brief')
-                try_load_manifest(content_loader, application, framework_data, 'briefs', 'briefs_search_filters')
+    # replace placeholder _content_loader_factory with properly initialized one
+    global _content_loader_factory
+    _content_loader_factory = _make_content_loader_factory(
```

Comment:
Monkey-patching?? :laughing: :see_no_evil:

Reply:
Well I'm monkey-patching my own function so I don't know if that counts.